### ISSUE TITLE
How to read an (image, label, segm) tuple from the training dataset with COCOInstance?

### ISSUE THREAD

**Issue by zeynepiskenderoglu** [boncey/Flickr4Java#677]
I'm having problems with gluoncv.

After the index is created in the document I left below
`train_image, train_label, train_segm = train_dataset[6]`
3 outputs can be obtained using this expression.

After i create index with dset.CocoDetection
`img, target = train_dataset[6]`
I can get output like this.

I have to fix this problem so I can continue with gluoncv.

[https://cv.gluon.ai/build/examples_instance/train_mask_rcnn_coco.html#data-loader](url)
